bitnami/wildfly/templates/NOTES.txt

```diff
@@ -1,13 +1,29 @@
 
 ** Please be patient while the chart is being deployed **
 
-{{- if .Values.ingress.enabled }}
+{{- if and .Values.exposeManagementConsole (or (eq "NodePort" .Values.service.type) (eq "NodePort" .Values.service.type) .Values.mgmtIngress.enabled) }}
+-------------------------------------------------------------------------------
+ WARNING
+
+    By specifying "exposeManagementConsole=true" you have most likely exposed
+    the WildFly Management console externally.
+
+    Please note this is not recommended for production environments since
+    you are exposing your WildFly server to potential attacks.
+-------------------------------------------------------------------------------
+{{- end }}
+
+{{- if or .Values.ingress.enabled .Values.mgmtIngress.enabled }}
 
 1. Get the Wildfly URL and associate its hostname to your cluster external IP:
 
    export CLUSTER_IP=$(minikube ip) # On Minikube. Use: `kubectl cluster-info` on others K8s clusters
    echo "Wildfly URL: http{{ if .Values.ingress.tls }}s{{ end }}://{{ .Values.ingress.hostname }}"
    echo "$CLUSTER_IP  {{ .Values.ingress.hostname }}" | sudo tee -a /etc/hosts
```

Comment:
I think this should be conditioned to `if .Values.ingress.enabled`, otherwise it will display false information when `ingress.enabled=false` and `mgmtIngress.enabled=true`.